.stylelintrc

```diff
@@ -5,10 +5,12 @@
     "stylelint-config-styled-components"
   ],
   "rules": {
+    "declaration-colon-newline-after": null,
     "declaration-empty-line-before": null,
     "keyframes-name-pattern": null,
     "value-keyword-case": ["lower", { ignoreKeywords: ['dummyValue'] }],
     "block-no-empty": null,
-    "number-leading-zero": null
+    "number-leading-zero": null,
+    "selector-type-no-unknown": null
```

Comment:
Does this remove the need for us to use the `/* sc-selector */` comments!? That's awesome

Reply:
I think so – probably why I didn't know about the https://www.styled-components.com/docs/tooling#interpolation-tagging 😮